source/includes/markdown/_07-auth.html.md

```diff
@@ -146,8 +146,11 @@ When using the `response_type=code`, your app will receive the following paramet
 | **code** | If response_type=code in the request, this is the code your app can exchange for a token |
 | **state** | The state parameter that was sent with the authorizing request |
 
-You should check that the state is the same in this response as it was in the request. You can read more about the `state`
-parameter [here](https://auth0.com/docs/protocols/oauth2/oauth-state).
+The `state` parameter is necessary to prevent CSRF attacks. As such, **you should check that the `state` is the same in this response as it was in the request.** 
```

Comment:
nit: I wonder if `must` is better than `should` here.

Reply:
`must`'d!